jumpscale/packages/threebot_deployer/chats/threebot.py

```diff
@@ -80,10 +81,6 @@ def payment_currency(self):
             required=True,
         )
 
-    @chatflow_step(title="3Bot version")
```

Comment:
the name of the step(function name)  needs to also be removed from the steps list